src/components/Agency/fullPage.server.tsx

```diff
@@ -124,6 +127,11 @@ jahiaComponent(
 			server.render.addCacheDependency({ node: imageNode }, renderContext);
 		}
 
+		const addresses = [{ address, id: currentNode.getIdentifier() }];
+		const addressItems: AddressItem[] = addresses
+			.filter((item): item is { address: string; id: string } => Boolean(item.address))
+			.map((item) => ({ label: item.address, ...item }));
```

Comment:
That's a lot of operations for just


```suggestion
		const addressItems = [{ address, label: address, id: currentNode.getIdentifier() }];
```

Did I miss something?

Reply:
oui l'adresse peut etre null et addressItems n'aime pas les adresse null

Reply:
mais on pourrait juste mettre un `if` aussi